javalib/src/main/scala/java/util/LinkedList.scala

```diff
@@ -0,0 +1,367 @@
+package java.util
+
+import scala.annotation.tailrec
+import scala.collection.JavaConversions._
+
+class LinkedList[E]() extends AbstractSequentialList[E]
+    with List[E] with Deque[E] with Cloneable with Serializable {
+
+  def this(c: Collection[_ <: E]) = {
+    this()
+    addAll(c)
+  }
+
+  import LinkedList._
+
+  private var head: Node[E] = null
+  private var last: Node[E] = null
+
+  private var _size = 0
+
+  def getFirst(): E =
+    if (_size <= 0)
+      throw new NoSuchElementException()
+    else peekFirst()
+
+  def getLast(): E =
+    if (_size <= 0)
+      throw new NoSuchElementException()
+    else peekLast()
```

Comment:
Fix what in the guide? It's all written, I believe. See https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md#braces and https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md#ifelse